core-web/libs/portlets/dot-ema/src/lib/components/edit-ema-language-selector/edit-ema-language-selector.component.scss

```diff
@@ -0,0 +1,22 @@
+@use "variables" as *;
+:host {
+    display: flex;
+}
+
+::ng-deep {
+    .p-overlaypanel.edit-ema-language-selector.p-component {
+        .p-overlaypanel-content {
+            padding: 0;
+
+            .p-listbox.p-component {
+                border: none;
+                border-radius: $border-radius-sm;
+
+                .p-listbox-header {
+                    border-top-left-radius: $border-radius-sm;
+                    border-top-right-radius: $border-radius-sm;
+                }
+            }
+        }
+    }
+}
```

Comment:
```suggestion
@use "variables" as *;

:host {
    display: flex;
    
    ::ng-deep {
        .p-overlaypanel.p-component {
            .p-overlaypanel-content {
                padding: 0;

                .p-listbox.p-component {
                    border: none;
                    border-radius: $border-radius-sm;

                    .p-listbox-header {
                        border-top-left-radius: $border-radius-sm;
                        border-top-right-radius: $border-radius-sm;
                    }
                }
            }
        }
    }
}
```

Reply:
This doesn't work because the overlay panel is appended to the body, that's why I added a class, to avoid the propagation of the styles

Reply:
The overlay have `appendTo` attr, not sure if works in this case, but worth to try no?

Reply:
I tried to append it to the button, but it got way misplaced